InterGalacticCurrencyConvertor/src/main/java/DataProcessResult.java

```diff
@@ -0,0 +1,44 @@
+package main.java;
+
+public class DataProcessResult{
```

Comment:
If you want to represent an empty result, you can create a static member variable:

public static DataProcessResult EMPTY_RESULT = new DataProcessResult(null, false);

Reply:
updated